modelzoo/benchmark/README.md

```diff
@@ -0,0 +1,62 @@
+# Deeprec Benchmark
+The following is a brief directory structure and description for this example:
+
+```
+├── benchmark_result               # Output directory  
+│   ├── checkpoint                   # checkpoint file  
+│   ├── log                          # Log files  
+│   └── record                       # Training record  
+│       ├── script                     # Run model train.py in docker  
+│       └── tool                       # Benchmark auxiliary document  
+│           └── check_model.sh           # Check model name file  
+├── benchmark.sh                   # Benchmark script  
+├── config.yaml                    # Config file  
+├── log_process.py                 # Porcess log file and output reuslt  
+└── README.md                      # Directory structure and how to use this script
+```
+
+
+## Content
+- [Deeprec Benchmark](#deeprec-benchmark)
+  - [Content](#content)
+  - [Requirement](#requirement)
+  - [Usage](#usage)
+  - [Description of files](#description-of-files)
+    - [config.yaml](#configyaml)
+    - [log](#log)
+
+
+## Requirement
+The following test environment should be installed:
++ Docker : 20.10.12
++ Python : 3.6.8
++ Shyaml : 0.6.2
+
+## Usage
+1. `cd` to the directory `./benchmark`  
+2. modify the config files `config.yaml` to specify the parameters you want, including the test image, test model list, benchmark stock tf or not and parameters of deeprec benchmark.  
+3. run the script `bash benchmark.sh`  
+4. check log files in the directory `benchmark/benchmark_result/log/$CurrentTime`
+
+## Description of files
+### config.yaml
+- `deeprec_test_image` : the images used to benchmark deeprec,  the default value is `alibaba/deeprec-release-modelzoo:latest`
+
+- `tf_test_image` : the images used to benchmark stock tf, the default value is `alideeprec/deeprec-weekly-modelzoo:tf` 
+
+- `test model` : which models you need to benchmark, like `- WDL`. These models can be added or deleted according to the benchmark needs.  The default value is all six models that need to be benchmarked : `DLRM WDL DeepFM DSSM DIEN DIN`
+
+- `model_batchsize` : batchsize of all models, like `WDL: 512`. The corresponding value is set in the model. It is only displayed here and cannot be set.

```

Comment:
batch size of all models, like `WDL: 512`. Please keep the same batch size as used in model training. It is only used to calculate throughput and cannot be used to set training parameters.